types/test/fnull.ts

```diff
@@ -0,0 +1,13 @@
+import * as RA from 'ramda-adjunct';
+
+const fn = () => null;
```

Comment:
Let's define this function properly as a function receiving one argument that is being returned.
```js
const fn = v => v
```